woocommerce/woocommerce-android
# [Mobile Payments] Remove upsell banner from payment method selection screen

Closes: #8814


### Description

As JITM replaces hardcoded banner, this PR removes it from the Payment Method Selection Screen

### Testing instructions

* Start IPP payment
* Notice that the Payment Method Selection screen doesn't show upsell banner anymore
